shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz_hints/view/model/StepQuizHintsViewState.kt

```diff
@@ -0,0 +1,37 @@
+package org.hyperskill.app.step_quiz_hints.view.model
+
+// I think it would be better if we'll have StepQuizHintsViewState in scope of the Feature -> StepQuizHintsFeature.ViewState
+// Kotlin Extensions allows to do this?
```

Comment:
No, it's impossible to declare class as extension for another class. We can just declare `ViewState` inside `Feature`.